tutorials/scripting/scene_unique_nodes.rst

```diff
@@ -29,7 +29,7 @@ to its name in the scene tree:
 .. image:: img/percent.png
 
 To use a unique node in a script, use the ``%`` symbol and the node's
-name in the path for ``get_node()``. For example:
+name to create a path for ``get_node()``. For example:
```

Comment:
This sentence is badly phrased, and I don't think your suggestion helps, quite the opposite. How about:

> You can now use the node in your script. For example, you can reference it with
a ``get_node()`` method by typing the ``%`` symbol, followed by the node's name:

Reply:
I think it's clearer as far as technical correctness goes, but a rework is better. I like your suggestion, will apply it.